# Extract the best locations that entirely cover a map
# author: Tim Patten, t.patten@acfr.usyd.edu.au

uint8 tree_depth          # the tree depth for the search
float32 robot_height      # the rheight of the onboard sensor (assumes this is the top of the robot)
float32 robot_outer_range # the outer range limit of the onboard kinect sensor
float32 robot_inner_range # the inner range limit of the onboard kinext sensor
float32 robot_radius      # the radius of the robot
float32 step_size         # the step size to extract the initial exhaustive set of locations before the optimization
int8 maximum_iterations   # the maximum number of iterations allowed
bool visualize_on         # visualize the locations

string filename  # the filename to an octomap (.bt or .ot) or a directory containing a single octomap file
                 # if a directory is specified then the results will be saved in same directory for quick look up later on

---

# list of locations (no specific order)
geometry_msgs/Point[] positions
